idunn/directions/models.py

```diff
@@ -198,6 +201,18 @@ def __init__(self, **data):
                     features_list.append(feature)
             data["geometry"] = {"type": "FeatureCollection", "features": features_list}
 
+        if "start_time" not in data or "end_time" not in data:
+            if "dTime" in data and "aTime" in data:
+                # Handle combigo's output format
+                start = datetime.utcfromtimestamp(int(data["dTime"]) / 1000)
+                end = datetime.utcfromtimestamp(int(data["aTime"]) / 1000)
+            else:
+                start = datetime.utcnow()
+                end = start + timedelta(0, data.get("duration", 0))
```

Comment:
Nitpick: 
```suggestion
                end = start + timedelta(seconds=data.get("duration", 0))
```
looks clearer to me.